headers/MainMenu.h

```diff
@@ -1,38 +1,27 @@
 #pragma once
 #include "LTexture.h"
 #include "Button.h"
+#include "MainMenuTab.h"
 
 class Game;
 
-class MainMenu : ButtonObject, Renderable
+class MainMenu : public MainMenuTab, ButtonObject
```

Comment:
Sicher nicht "public ButtonObject?
```suggestion
class MainMenu : public MainMenuTab, public ButtonObject
```